monai/data/png_writer.py

```diff
@@ -76,6 +76,10 @@ def write_png(
         else:
             raise ValueError(f"Unsupported scale: {scale}, available options are [255, 65535]")
 
+    # PNG data must be int number
+    if data.dtype != np.uint8 and data.dtype != np.uint16:
```

Comment:
this is a good comment? https://deepsource.io/gh/Project-MONAI/MONAI/run/c6544bdf-660c-4bd1-8192-f184f59f7f0f/python/PYL-R1714

Reply:
Hi @wyli ,

I used `data.dtype in (np.uint8, np.uint16)` before, but it can't pass `mypy`. Let me add a ignore.

Thanks.